configure.ac

```diff
@@ -42,8 +44,10 @@ PKG_CHECK_MODULES(libqb, [libqb])
 CPPFLAGS="$CPPFLAGS -Werror"
 if test $HAVE_pacemaker = 0 -a $HAVE_pcmk = 0; then
     AC_MSG_ERROR(No package 'pacemaker' found)
+elif test $HAVE_cmap = 0; then
+    AC_MSG_ERROR(No package 'cmap' found)
```

Comment:
I don't think corosync 1 has libcmap. sbd supports corosync 1 + pacemaker plugin. We don't have to implement the new behavior for the old stack, but we shouldn't break it